code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -352,9 +359,26 @@
 	R.my_atom = src.emag
 	R.add_reagent("beer2", 50)
 	src.emag.name = "Mickey Finn's Special Brew"
-	return
-
 
+/obj/item/weapon/robot_module/butler/recharge_consumable(var/mob/living/silicon/robot/R)
```

Comment:
I copied that from the engieborg code, worked fine from testing, initially confused it with respawn_consumeable.
Also noticed a very similar thing in the mommi code, if nothing else it works.